.github/workflows/line-length-check.yml

```diff
@@ -0,0 +1,115 @@
+name: Line Length Check
+
+on:
+  pull_request:
+    branches: [ '*' ]
+
+jobs:
+  line-length-check:
+    runs-on: ubuntu-latest
+    name: Check 80 character line limit
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: Check line length in PR changes
+      run: |
+        # Get the base branch (usually main/master)
+        BASE_BRANCH="${{ github.event.pull_request.base.ref }}"
+
+        echo "Checking line length (max 80 characters) for changed files in " \
+          "src/, examples/, and jni/ directories..."
```

Comment:
Should read "... and native/ directories...". wolfcrypt-jni uses the "jni" directory for native sources, but wolfssljni uses the "native" directory name instead.